app/models/region.rb

```diff
@@ -135,8 +135,9 @@ def syncable_patients
     case region_type
```

Comment:
`def syncable_patient_ids`? `syncable_patients` could just `Patient.where(id: syncable_patient_ids)` like in the previous PR.